src/coreclr/tools/Common/TypeSystem/Common/TypeDesc.TypeEquivalence.cs

```diff
@@ -136,20 +136,20 @@ public bool TypeHasCharacteristicsRequiredToBeLoadableTypeEquivalentType
                             return false;
                     }
 
-                    foreach (var field in GetFields())
+                    foreach (var fieldInfo in GetFields())
```

Comment:
I forget where we landed. `field` is now considered a keyword in any property accessor, such that trying to use it as a local variable name is an error?